game-core/src/main/java/games/strategy/triplea/settings/ClientSetting.java

```diff
@@ -211,13 +205,26 @@ public static void flush() {
     }
   }
 
+  private static Preferences getPreferences() {
+    return Optional.ofNullable(preferencesRef.get())
+        .orElseThrow(() -> new IllegalStateException("ClientSetting framework has not been initialized. "
+            + "Did you forget to call ClientSetting#initialize() in production code "
+            + "or ClientSetting#setPreferences() in test code?"));
+  }
```

Comment:
this is a cleanup of method on line 180, and also moved it to be below its first line of usage.